### ISSUE TITLE
Namada service fails during the syncing process

### ISSUE THREAD

**Issue by timagixe** [obajay/nodes-Guides#17]
Hello @obajay! First of all thank you for the details guides. They are really helpful!

I was following this guide ([Namada Guide](https://github.com/obajay/nodes-Guides/blob/main/Projects/Namada/README.md)) in order to start running namada validator.

When I reached this step: [Start Node](https://github.com/obajay/nodes-Guides/blob/main/Projects/Namada/README.md#-start-node)

Everything was OK, my node was syncing, the output of this command `curl -s localhost:26657/status` was `true`, so the node was catching up.

Some time later when running this command `sudo journalctl -u namadad -f -o cat`, I spotted this in the logs:

```
I[2024-01-06|09:20:57.933] deprecated usage found in configuration file usage="[fastsync] table detected. This section has been renamed to [blocksync]. The values in this deprecated section will be disregarded."
I[2024-01-06|09:20:57.933] deprecated usage found in configuration file usage="fast_sync key detected. This key has been renamed to block_sync. The value of this deprecated key will be disregarded."
I[2024-01-06|09:20:57.953] service start                                module=proxy msg="Starting multiAppConn service" impl=multiAppConn
I[2024-01-06|09:20:57.953] service start                                module=abci-client connection=query msg="Starting socketClient service" impl=socketClient
E[2024-01-06|09:20:57.954] abci.socketClient failed to connect to tcp://127.0.0.1:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp 127.0.0.1:26658: connect: connection refused"


thread 'main' has overflowed its stack
fatal runtime error: stack overflow
namadad.service: Main process exited, code=dumped, status=6/ABRT
namadad.service: Failed with result 'core-dump'.
namadad.service: Consumed 1.541s CPU time.
```

@obajay do you have any ideas how to fix this?

**Comment by timagixe** [comment]
@obajay here are libs versions I am using:

```
# cometbft version
0.37.2

# protoc --version
libprotoc 24.4

# namada --version
Namada v0.28.1
```

I see there were [0.28.2](https://github.com/anoma/namada/releases/tag/v0.28.2) and [0.29.0](https://github.com/anoma/namada/releases/tag/v0.29.0) version of namada already released.

0.28.2 incldues fixes for DB to avoid stack overflow issue for node.

In my case, seems like this is the issue:

```
thread 'main' has overflowed its stack
fatal runtime error: stack overflow
```

Should I use latest namada version 0.29.0 or at least 0.28.2? WDYT?

**Comment by timagixe** [comment]
The issue is solved after I updated NAMADA_TAG to 0.28.2 which includes fix for stack overflow issue.

`# curl -s localhost:26657/status | jq .result.sync_info` results in:

```
{
  "latest_block_hash": "04757BA5C730EB5D562300A8A4651B93BAA6EA6A3BD3C11DEEC0EA4686A50B88",
  "latest_app_hash": "793BBA04E761D28E214D62A6E45E9382B351C6F09F48085056B6667D5E7B16C7",
  "latest_block_height": "39564",
  "latest_block_time": "2023-12-23T01:22:31.975681293Z",
  "earliest_block_hash": "F37D1F1A500EBA7BEE909EC4C3390AC4E9954BBB89CFCE0281F98BC4CF7DD7E3",
  "earliest_app_hash": "",
  "earliest_block_height": "1",
  "earliest_block_time": "2023-12-18T17:00:00Z",
  "catching_up": true
}
```
